Source/WebInspectorUI/UserInterface/Controllers/LayerTreeManager.js

```diff
@@ -197,6 +197,21 @@ WI.LayerTreeManager = class LayerTreeManager extends WI.Object
         });
     }
 
+    snapshotForLayer(layer, callback)
+    {
+        console.assert(this.supported);
+
+        let target = WI.assumingMainTarget();
+        target.LayerTreeAgent.requestContent(layer.layerId, function(error, content) {
+            if (error) {
+                console.error("Failed to get layer snapshot:", error);
```

Comment:
lets use `WI.reportInternalError` so that it actually shows an error in local builds (i.e. more likely that someone sees it and reports it and we can fix it)